src/gateway.rs

```diff
@@ -106,7 +106,7 @@ impl Gateway {
             messages,
             beacon_handler,
             listen_address: settings.listen.clone(),
-            udp_runtime: UdpRuntime::new(&settings.listen).await?,
+            udp_runtime: UdpRuntime::new(&settings.listen).await.map_err(Box::new)?,
```

Comment:
do you actually need this? the `from` in the error definition doesn't do this for you already?

Reply:
Yes. Errors otherwise.

The from can only convert from the exact type. If it was used in multiple places I'd manually implement From to do the boxing, but this is the only spot